src/infra/pg-secrets.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: pg-secrets
+type: Opaque
+data:
+  POSTGRES_USER: cG9zdGdyZXM=
+  POSTGRES_PASSWORD: c3lzdGVt
+  POSTGRES_URL: amRiYzpwb3N0Z3Jlc3FsOi8vcG9zdGdyZXM6NTQzMi9wb3N0Z3Jlcz9jdXJyZW50U2NoZW1hPWxhd19vZmZpY2Vfc2NoZW1h
```

Comment:
maybe is not a good idea to env the whole DB URL. I recommend you to env only db host.